actionpack/lib/action_controller/metal/rendering.rb

```diff
@@ -39,13 +39,7 @@ def render(*args) #:nodoc:
     # Overwrite render_to_string because body can now be set to a Rack body.
     def render_to_string(*)
```

Comment:
As described the above comment line, responding to `each` is a Rack spec, that is not only Array but also File etc.

https://github.com/rack/rack/blob/master/SPEC.rdoc#label-The+Body

Reply:
I see! Thank you for the context.